cocos/renderer/backend/opengl/CommandBufferGL.h

```diff
@@ -190,6 +190,7 @@ class CommandBufferGL final : public CommandBuffer
     bool _generatedFBOBindColor = false;
     bool _generatedFBOBindDepth = false;
     bool _generatedFBOBindStencil = false;
+    bool _framebufferReadWriteDisabled = false;
```

Comment:
Better to check for Windows and Linux here also to avoid not used variable for other platforms and probably warning.

Reply:
Got it.